doc/src/concepts/proxies.md

```diff
@@ -0,0 +1,25 @@
+# Proxies
+
+`rustup` provides a number of wrappers for common Rust tools.
+These are called _proxies_ and represent commands which are
+provided by the various [components][].
+
+The list of proxies is currently static in `rustup` and is as follows:
+
+[components]: concepts/components.html
+
+- `rustc` is the compiler for the Rust programming language, provided by the project itself and come from the `rustc` component.
+
+- `rustdoc` is a tool distributed in the `rustc` component which helps you to generate documentation for Rust projects.
+
+- `cargo` is the Rust package manager which downloads your Rust package’s dependencies, compiles your packages, makes distributable packages, and uploads them to crates.io and the Rust community’s package registry and it comes from the `cargo` component.
+
+- `rust-lldb` and `rust-gdb` are simple wrappers around the `lldb` and `gdb` debuggers respectively. The wrappers enable some pretty-printing of Rust values and add some convenience features to the debuggers by means of their scripting interfaces.
+
+- `rls` and `rust-analyzer` are two different generations of IDE integration tooling. They implement the language-server protocol to permit IDEs and editors such as Visual Studio Code, ViM, or Emacs, access to the semantics of the Rust code you are editing. They come from their respectively named components.
+
+- `cargo-clippy` is a lint tool which provides extra checks for common mistakes and stylistic choices and it comes from the `clippy` component.
+
+- [clippy-driver](https://github.com/rust-lang/rust-clippy/blob/master/src/driver.rs) is a driver that gives description of the provided lint with an option and also checks a package and it comes from the `clippy` component.
```

Comment:
I think `clippy-driver` is actually semi-deprecated so perhaps combine it into the `cargo-clippy` line with:

```markdown
- `cargo-clippy` and `clippy-driver` are related to the `clippy` linting tool which ...
```